src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -60,7 +62,13 @@
        :label    (i18n/label :t/delete-for-me)
        :icon     :i/delete
        :id       :delete-for-me}])
-   (when (and (or outgoing can-delete-message-for-everyone?) config/delete-message-enabled?)
+   (when (and config/delete-message-enabled?
+              (cond
+                outgoing   true
+                community? (or can-delete-message-for-everyone-in-community?
+                               community-admin?)
```

Comment:
I believe can-delete-message-for-everyone-in-community? already contains logical that judge if user is admin in status-go side?

Reply:
Fixed